core/nodepool/config/config.go

```diff
@@ -64,13 +64,19 @@ func (c ProvidedConfig) StackConfig(opts StackTemplateOptions) (*StackConfig, er
 		return nil, fmt.Errorf("failed to generate config : %v", err)
 	}
 
-	compactAssets, err := cfg.ReadOrCreateCompactTLSAssets(opts.TLSAssetsDir, cfg.KMSConfig{
-		Region:         stackConfig.ComputedConfig.Region,
-		KMSKeyARN:      c.KMSKeyARN,
-		EncryptService: c.providedEncryptService,
-	})
+	if stackConfig.ComputedConfig.IsChinaRegion {
```

Comment:
If `IsChinaRegion` is almost meant for `not TLSAssetsEncryptionEnabled` in the context of this PR,
can we introduce new func like:

```golang
(c *Cluster) TLSAssetsEncryptionEnabled() bool {
    return !c.IsChinaRegion
}
```

and refer to it from here and various locations including templates?